config/environments/test.rb

```diff
@@ -9,7 +9,7 @@
   # Settings specified here will take precedence over those in
   # config/application.rb.
 
-  config.cache_classes = true
+  config.cache_classes = false
```

Comment:
Why this change? You're not reloading code during a test run

Reply:
I thought I had a Rails warning about it but I'll revert.